src/collective/sidebar/controlpanel/controlpanel.py

```diff
@@ -62,6 +62,21 @@ class IControlPanel(Interface):
         default=False,
     )
 
+    icon_font = schema.Choice(
+        title=_(u'controlpanel_sidebar_choose_icon_font_title',
+                default=u'Choose Icon Font'),
+        description=_(
+            u'controlpanel_sidebar_choose_icon_font_description',
+            default=(
+                u'When you installed a different icon font, you can tell '
+                u'sidebar here to use it. For Fontello we assume you set '
+                u'"icon" as font name')
+        ),
+        values=('Glyphicons', 'Fontello', 'Font Awesome', 'Font Awesome Pro', 'Font Awesome Light'),  # noqa: 501
```

Comment:
Font Awesome and Font Awesome Pro can be combined somehow, since they both use the `fa` prefix.